frontend/src/core/comments/components/AddComment.js

```diff
@@ -0,0 +1,69 @@
+/* @flow */
+
+import * as React from 'react';
+import { Localized } from '@fluent/react';
+
+import './Comment.css';
+
+import { UserAvatar } from 'core/user'
+
+type Props = {|
+    user: string,
+    username: string,
+    imageURL: string,
+    translationId: number,
+    addComment: (string, number) => void,
+|};
+
+
+export default function AddComments(props: Props) {
+    const {
+        user,
+        username,
+        imageURL,
+        translationId,
+        addComment,
+    } = props;
+
+    let commentInput: any = React.createRef();
+
+    if (!user) {
+        return null;
+    }
+
+    const submitComment = (event: SyntheticKeyboardEvent<>) => {
+        event.preventDefault();
+        const comment = commentInput.current.value;
+
+        if(!comment) {
+            return null;
+        }
+
+        addComment(comment, translationId);
+        commentInput.current.value = '';
+    };
+
+    return <div className='comment add-comment'>
+        <UserAvatar
+            user={ user }
+            username={ username }
+            title=''
+            imageUrl={ imageURL }
+        />
+        <form className='container' onSubmit={ submitComment }>
+            <Localized
+                id='comments-AddComment--input'
+                attrs={{ placeholder: true }}
+            >
+                <input
```

Comment:
The specs indicate that this will have an [autogrow](https://goodies.pixabay.com/jquery/auto-grow-input/demo.html) feature which would apply to an input field. As per @mathjazz's [previous comment](https://github.com/mozilla/pontoon/pull/1524#issuecomment-573549432), this will be added in a new PR.

Reply:
Note that the spec links to a jQuery lib, which is no longer useful for us now that we switched to React. I'll update the spect with a link to https://www.npmjs.com/package/react-textarea-autosize.